docs_src/design/adr/security/0017-consul-security.md

```diff
@@ -0,0 +1,80 @@
+# Securing access to Consul
+
+## Status
+
+** proposed **
+
+## Context
+
+This ADR defines the motiviation and approach used to secure access
+to the Consul component in the EdgeX archticture.
+
+Consul provides several services for the EdgeX architecture:
+
+- Service registry
+- Service health monitoring
+- Mutable configuration data
+
+Use of the services provided by Consul is optional on a service-by-service basis.
+Use of the registry is controlled by the `--registry` flag provided to an EdgeX service.
```

Comment:
Minor nit... all of our service command line options accept `-` or `--`. There's also a long and short form of each flag:

 * `-r | -registry`
 * `-cp | --configProvider`
 *

Reply:
Fixed